src/main/java/org/meteordev/starscript/StandardLib.java

```diff
@@ -166,4 +172,44 @@ public static Value pad(Starscript ss, int argCount) {
 
         return Value.string(new String(padded));
     }
+
+    public static Value formatDateTime(Starscript ss, int argCount) {
+        if (argCount != 1) ss.error("formatTime() requires 1 argument, got %d.", argCount);
+        try {
+            String fmt = ss.popString("Argument to formatTime(fmt) needs to be a string.");
+            SimpleDateFormat formatter = new SimpleDateFormat(fmt);
+            return Value.string(formatter.format(new Date()));
+        }
+        catch (IllegalArgumentException e) {
+            ss.error(e.toString());
+        }
+        return Value.null_();
+    }
+
+    public static Value format(Starscript ss, int argCount) {
+        if (argCount < 1) ss.error("format(fmt, ...args) requires at least 1 argument, got %d.", argCount);
+        ArrayList<Object> args = new ArrayList<Object>();
+        for (int i = 1; i < argCount; i ++) {
+            Value v = ss.pop();
+            Object o = null;
+            switch (v.type) {
+                case Null: o = Value.null_(); break;
```

Comment:
this is wrong, should just be `o = null`

Reply:
> this is wrong, should just be `o = null`

iirc I used this because I saw `Value.null_()` being used somewhere else. Anyways do you want me to edit and fix this, if you are going to accept this pr?

Reply:
For me personally it would be really nice, if this PR is being merged. :)

Reply:
> > this is wrong, should just be `o = null`
> 
> iirc I used this because I saw `Value.null_()` being used somewhere else. Anyways do you want me to edit and fix this, if you are going to accept this pr?

`Value.null_()` is the starscript null type, but in this case you’re feeding it into the java `String.format()` method, which takes in the java `null`

Reply:
Check my new commits in case you're interested in accepting this PR. Should be fixed now.